db/data_migration/20160202115027_publish_classification_placeholders.rb

```diff
@@ -0,0 +1 @@
+Classification.where.not(content_id: nil).each(&:publish_to_publishing_api)
```

Comment:
FYI, there's a [rake task that does this](https://github.com/alphagov/whitehall/blob/master/lib/tasks/publishing_api.rake#L42), so we could potentially skip this altogether...